apps/web-giddh/src/app/settings/personal-information/personal-information.component.html

```diff
@@ -0,0 +1,86 @@
+<div>
+    <div class="row">
+        <!-- <div class="col-md-12 mt-15">
+            <div class="d-flex form-group">
+                <div class="custom-file cp" id="customFile">
+                    <input type="file" class="custom-file-input" id="exampleInputFile" aria-describedby="fileHelp">
+                    <label class="custom-file-label d-flex align-items-center justify-content-center"
+                        for="exampleInputFile">
+                        <i class="icon-plus"></i>
+                    </label>
+                </div>
+                <div class="font-14 pl-2">
+                    <p class="pb-1">This logo will appear on the vouchers (estimates, invoices, etc.) that are created.
+                    </p>
+                    <p class="pb-1 img-size">Preferred Image Size: 240px x 240px @ 72 DPI Maximum size of 1MB.</p>
+                    <a href="#">Remove Logo</a>
+                </div>
+            </div>
+        </div> -->
```

Comment:
@prateek-giddh Where is this code written? I couldn't find remove logo button and text